packages/@storefront/flux-capacitor/src/core/actions/creators.ts

```diff
@@ -1159,6 +1159,15 @@ namespace ActionCreators {
     return createAction(Actions.UPDATE_LOCATION, location);
   }
 
+  /**
+   * Set the sessionId in the store to the given id.
+   * @param sessionID- The sessionId to set.
```

Comment:
Misc.
- Param name should be `sessionId` (not `sessionID`).
- Space between `sessionId` and trailing `-`.

Reply:
Done.